webpack/README.md

```diff
@@ -28,6 +28,10 @@ $ npm start
 After this you can visit http://localhost:8080 and you should see an interactive
 markdown renderer!
 
+> **Note**: Running this example in a web browser requires multi-value support
+> for wasm in a browser. At the time of this writing only Chrome has support for
+> multi-value behind a flag.
```

Comment:
Perhaps "At the time of this writing only Chrome has support for multi-value, and it has to be enabled by activating the "Experimental WebAssembly" setting under `chrome://flags`"?